saleor/plugins/webhook/plugin.py

```diff
@@ -2036,15 +2050,34 @@ def product_variant_stocks_updated(
             )
         return previous_value
 
+    def _get_webhooks_for_checkout_events(
```

Comment:
Suggestion: do we need `_get_webhooks_for_checkout_events` and `_get_webhooks_for_order_events` at all? Both just calls `_get_webhooks_for_channel_events` with the channel from checkout/order, seems like they are not needed.

Reply:
@IKarbowiak I've added it only, to wrap the helper logic inside the re-usable function that I can use directly with the checkout, and directly with the order. 
If you don't like it, I can drop this, and directly use the helper :)